en/reference/validation-overrides.html

```diff
@@ -75,3 +73,101 @@ <h2 id="allow">allow</h2>
 </tr>
   </tbody>
 </table>
+
+
+<h2 id="list">List of validation overrides</h2>
+
+<table>
+  <tr>
+    <th>Validation id</th>
```

Comment:
```suggestion
    <th>ID</th>
```

I would have suggested "Validation Override ID" for clarity, but that's far to long for a single column.